redaxo/src/core/update.php

```diff
@@ -96,6 +96,13 @@
         ->update();
 }
 
+if (rex_string::versionCompare(rex::getVersion(), '5.6.2', '<')) {
```

Comment:
Müsste `<=` sein, oder?
Wobei ich `'5.7-beta1', '<'` bevorzugen würde (oder theoretisch auch mit `-dev`).

Reply:
Müssten wir so oder so weiterhin beobachten, da wir noch nicht wissen in welcher Version die Minibar veröffentlicht wird. Sollte man hier in der Entwicklung eher mit Platzhalter arbeiten und das Release Skript passt das an? (Im Allgemeinen für Coreentwicklungen)

Habe es auf 5.7-dev angepasst 80c30e76f64e597b57d0b638ceff2de40a97ba21

Reply:
ich würde keinen automatismus schaffen. das ist so einfacher.